config/locales/es.yml

```diff
@@ -0,0 +1,11 @@
+es:
+  ibandit:
+    invalid_country_code: "'%{country_code}' no es un código de país ISO 3166-1 válido"
+    invalid_check_digits: "Los dígitos de control no han pasado la verificación de módulo. Se esperaba: '%{expected_check_digits}', recibido '%{check_digits}'."
+    invalid_length: "La longitud no casa con la especificación SWIFT (Se esperaba %{expected_length} símbolos, se ha recibido %{length})"
+    is_required: "es obligado"
+    wrong_length: "esta de longitud equivocado (debería formarse de %{expected} símbolos)"
```

Comment:
"está"

Also not sure if "está de" or "tiene el" would sound more natural. @alan?